docker_bitnami/prebuildfs/opt/bitnami/scripts/libbitnami.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+
+# Copyright (c) 2015-2021 Bitnami
```

Comment:
Are we aware of mixing licenses in src tree? And its consequences?

Reply:
Aware of mixing, but not sure about the consequences? If using the external files is not possible, could probably replace them with instruction where to find them, e.g here: https://github.com/bitnami/bitnami-docker-redis-cluster/tree/master/5.0/debian-10